newdle/client/src/components/DurationPicker.module.scss

```diff
@@ -0,0 +1,13 @@
+@import '../styles/palette.scss';
+
+.duration-picker {
+  background-color: $pink;
+  display: inline-block;
+  padding: 5px;
+  color: $white;
+
+  .dropdown {
+    min-width: 0 !important;
+    border-radius: 0 !important;
```

Comment:
Are you sure there's no alternative to these `!important`?

Reply:
Do we need an alternative in this case? It's only class that's going to be used inside the duration-picker.